### ISSUE TITLE
Definition  of all_stages_loss, how to choose suitable coefficient for separate stage loss?

### ISSUE THREAD

**Issue by zwlshine** [JohnleeHIT/Brats2019#8]
Definition  of all_stages_loss is:        self.all_stages_loss = 0.4 * self.total_wght_loss +  0.8 * self.stage2_loss + self.stage3_loss, why use 0.4、0.8、1 as the three stage's loss coefficient?

**Comment by JohnleeHIT** [comment]
I tried several different weighting parameters,  it turns out that there exists no big difference,  details can be found in my paper.

**Comment by zwlshine** [comment]
@JohnleeHIT 
These weighting parameters really have big influence on dice of segmentation.
